app/services/bill-runs/two-part-tariff/review-bill-run.service.js

```diff
@@ -51,12 +53,12 @@ async function go (id, page, yar) {
 
 function _getFilters (id, yar) {
   const filters = yar.get(`review-${id}`)
-
   const filterIssues = filters?.filterIssues
   const filterLicenceHolderNumber = filters?.filterLicenceHolderNumber
   const filterLicenceStatus = filters?.filterLicenceStatus
+  const filterProgress = filters?.filterProgress
 
-  return { filterIssues, filterLicenceHolderNumber, filterLicenceStatus }
+  return { filterIssues, filterLicenceHolderNumber, filterProgress, filterLicenceStatus }
```

Comment:
With our standard of putting things in alphabetical order, I think we need to re-arrange this
```suggestion
  return { filterIssues, filterLicenceHolderNumber, filterLicenceStatus, filterProgress }
```